models/example_model.py

```diff
@@ -7,18 +7,16 @@
 class ExampleModel(BaseModel):
     def __init__(self, config):
         super(ExampleModel, self).__init__(config)
-        self.build_model()
-        self.init_saver()
         self.max_sequence_length = config.max_sequence_length
         self.embedding_dim = config.embedding_dim
         self.vocab_size = config.vocab_size
-        self.callbacks = []
+        self.build_model()
 
-    def get_embedding_matrix():
+    def get_embedding_matrix(self):
         # Todo: implement by gammal
         return None
         
-    def embedding_layer():
+    def embedding_layer(self):
         embedding_matrix = get_embedding_matrix()
```

Comment:
Shouldn't this be self.get_embedding_matrix() ?

Reply:
yup